tools/workflow/clean_aws_rsc.go

```diff
@@ -0,0 +1,167 @@
+		describeInstancesOutput, describeInstanceErr := ec2client.DescribeInstances(&describeInstancesInput)
+		if describeInstanceErr != nil {
+			log.Printf("Failed to get instance for error %v", describeInstanceErr)
+			return describeInstanceErr
+		}
+		for _, reservation := range describeInstancesOutput.Reservations {
+			for _, instance := range reservation.Instances {
+				//only delete instance if older than 5 days
+				if time.Now().UTC().Add(-1 * time.Hour * 24 * pastDayDelete).After(*instance.LaunchTime) {
+					log.Printf("Try to delete instance %s tags %v launch-date %v", *instance.InstanceId, instance.Tags, instance.LaunchTime)
+					deleteInstanceIds = append(deleteInstanceIds, instance.InstanceId)
+				}
+			}
+		}
+		if describeInstancesOutput.NextToken == nil {
+			break
+		}
+		nextToken = describeInstancesOutput.NextToken
+	}
+
+	if len(deleteInstanceIds) < 1 {
+		log.Printf("No instances to delete")
+		return nil
+	}
+
+	terminateInstancesInput := ec2.TerminateInstancesInput{InstanceIds: deleteInstanceIds}
+	_, terminateInstancesErr := ec2client.TerminateInstances(&terminateInstancesInput)
+	if terminateInstancesErr != nil {
+		log.Printf("Failed to terminate instances %v because of %v",deleteInstanceIds, terminateInstancesErr)
+		return terminateInstancesErr
+	}
+
+	return nil
+}
+
+func destroyLoadBalancerResource()(error){
+	// Set up aws go sdk session
+	//Enable default region and credentials
+	//Documents: https://docs.aws.amazon.com/ja_jp/sdk-for-go/v1/developer-guide/configuring-sdk.html
+	session := session.Must(
+		session.NewSessionWithOptions(
+			session.Options{
+				SharedConfigState: session.SharedConfigEnable,
+			}))
+
+	svc := elbv2.New(session)
+
+	//ELB Go SDK currently does not support filter tag or filter wildcard. Only supports with matching name
+	//Documentation: https://github.com/aws/aws-sdk-go/blob/02266ed24221ac21bb37d6ac614d1ced95407556/service/elbv2/api.go#L5879-L5895
+
+	describeLoadBalancerInputs := &elbv2.DescribeLoadBalancersInput{}
+
+
+	describeLoadBalancerOutputs, describeLoadBalancerErr := svc.DescribeLoadBalancers(describeLoadBalancerInputs)
+
+	if describeLoadBalancerErr != nil {
+		log.Printf("Failed to get metadata for load balancer because of %v", describeLoadBalancerErr)
+		return describeLoadBalancerErr
+	}
+
+	for _, lb := range describeLoadBalancerOutputs.LoadBalancers {
+
+		//Skipping lb that does not contain aoc-lb string (relating to aws-otel-test-framework)
+		if filterLbNameResult := strings.Contains(*lb.LoadBalancerName, containLbString); !filterLbNameResult {
+			log.Printf("Skipping lb %s because of not containing %s", *lb.LoadBalancerName, containLbString)
+			continue
+		}
+
+		//Skipping lb that does not older than 5 days
+		if !time.Now().UTC().Add(-1 * time.Hour * 24 * pastDayDelete).After(*lb.CreatedTime) {
+			log.Printf("Skipping lb %s with launch-date %v", *lb.LoadBalancerName, lb.CreatedTime)
```

Comment:
This and the other skipped logging seem like they'll overwhelm the logging. I don't think we need to log every single load balancer we have. Just the ones we delete.